addons/web/static/tests/mobile/burger_menu_tests.js

```diff
@@ -122,3 +122,37 @@ QUnit.test("Burger menu closes when an action is requested", async (assert) => {
     assert.containsNone(document.body, ".o_burger_menu");
     assert.containsOnce(document.body, ".o_kanban_view");
 });
+
+QUnit.test("Burger menu closes when click on menu item", async (assert) => {
+    assert.expect(4);
```

Comment:
no need for expect. Use it when there is a chance asserts are skipped (like in a mockRPC). 
And yes, all the code based has this function but that's just bad practice copied/pasted.

Reply:
@SimonGenin done.Thanks